Library/Formula/mu.rb

```diff
@@ -29,10 +29,6 @@ def install
     # shipped by default with Mac OS X is too old.
     ENV["EMACS"] = "no" if build.without? "emacs"
 
-    # https://github.com/djcb/mu/issues/380
-    # https://github.com/djcb/mu/issues/332
-    ENV.O0 if MacOS.version >= :mavericks && ENV.compiler == :clang
```

Comment:
The test checks for this. Before this version, the test would fail if you didn't do this.

It passes happily now.